### PR TITLE
fixed error of js errors in console

### PR DESCRIPTION
**Pull request by Subhang23**
Pull Request for Issue #27606 .

### Summary of Changes

Since the the eventListener might be used later I have wrapped it in a try catch block

### Expected result
There to be no error in the console

### CODE REVIEW

**Comment by Subhang23** [comment]
@Quy is this what was intended?

**Comment by Fedik** [comment]
sorry, the fix are wrong,
to make a correct fix you have to analyse what cause an error, first

File: build/media_source/layouts/js/joomla/html/batch/batch-language.es6.js [hunk 1]
```diff
@@ -37,7 +37,10 @@
     }
 
     if (batchCopyMove) {
-      batchSelector.addEventListener('change', onChange);
```
Comment (dgrammatiko) [inline build/media_source/layouts/js/joomla/html/batch/batch-language.es6.js:40]:
```js
if (batchCopyMove && batchSelector) {
```

**Review by dgrammatiko — CHANGES_REQUESTED** [review]

**Comment by SharkyKZ** [comment]
Fields have their own batch script https://github.com/joomla/joomla-cms/blob/0a12b92a93630435f7ca0c5129f48a11d875ed57/build/media_source/com_fields/js/admin-fields-default-batch.es6.js. We can either move relevant part of the code to `batch-language.js` and remove that script or start loading `batch-language.js` only where it's actually used.

**Comment by Quy** [comment]
Please test PR #31791

**Pull request closed without merging**